apps/site/assets/ts/stop/components/StopPageRedesign.tsx

```diff
@@ -58,7 +58,11 @@ const StopPageRedesign = ({
   // Get only alerts that are current
   const currentAlerts = filter(
     alertsForStopResult.data.concat(routeWideAlertsArray),
-    alert => isCurrentAlert(alert)
+    alert =>
+      isInNextXDays(alert, 7) &&
+      ["suspension", "stop_closure", "station_closure", "shuttle"].includes(
```

Comment:
I'd break this filtering out into a separate line/step. Its doing to much filtering by date and effect